packages/markdown-slate/src/SlateTransformer.js

```diff
@@ -51,24 +52,39 @@ class SlateTransformer {
             object: 'value',
             document: parameters.result
         };
+        const paragraphSpaceNodeJSON = {
+            object: 'block',
+            type: 'paragraph',
+            data: {
+            },
+            nodes: [
+                {
+                    object: 'text',
+                    text: '',
+                    marks: []
+                }
+            ]
+        };
+
+        // Find any clauses next to each other, force in a paragraph between
+        const newArray = [];
+        result.document.nodes.forEach((node, i) => {
+            newArray.push(node);
+            if (node.type === CLAUSE || result.document.nodes[i + 1] === CLAUSE) {
```

Comment:
- What happens on the last node with `i+1` ?
- Should this compare against the type for the next node as well?

Reply:
Another pretty bad mistake, I think I should have made this `&&` rather than `||`.

I should change this to a traditional for loop to handle only going to `array.length - 1`